static/app/utils/useUserTeams.tsx

```diff
@@ -26,7 +26,7 @@ function buildUserTeamsQueryKey(orgSlug: string): ApiQueryKey {
  * const {teams, isLoading, isError} = useUserTeams();
  * ```
  */
-export function useUserTeams(): UseTeamsResult {
+export function useUserTeams(loadAllTeams?: boolean): UseTeamsResult {
```

Comment:
question, but what teams does this return by default if you don't use loadAllTeams? When we expose interfaces like this, it is confusing as to how to properly use them, because eod, why wouldn't you want it to return all teams?

What would be the downside of making this a default behavior inside the hook like we do with the `superuser` check, instead of relying on the caller to pass the correct argument? My reasoning for that would be that an `org:admin` role is roughly equivalent to a `superuser` role from a user perspective, so the logic should be the same in both instances instead of relying on the user to pass the correct argument.

Reply:
That makes sense. I didn't move the logic inside originally b/c I wasn't sure where else `useUserTeams` was used, but it was only used in 1 other place in the alert rules tab. The logic doesn't affect that, so should be good to default to the behavior